server/lib/oauth2/oauth2.deleteVar.js

```diff
@@ -0,0 +1,31 @@
+const logger = require('../../utils/logger');
+
+/**
+ * @description Delete client_id and secrei_id of current oauth2 integration.
+ * @param {string} integrationName - Name of oauth2 integration.
+ * @param {string} serviceId - Gladys serviceId of current integration.
+ * @param {string} userId - Gladys userId of current session.
+ * @returns {Promise} Resolve with current integration service id.
+ * @example
+ * oauth2.deleteVar(
+ *  integrationName: 'withings',
+ *  userId: fd81vs687f0bf3414e0fe3facfba7be9455109409a'
+ *  }
+ * );
+ */
+async function deleteVar(integrationName, serviceId, userId) {
+  try {
+    await this.gladys.variable.destroy(`${integrationName.toUpperCase()}_CLIENT_ID`, serviceId, userId);
```

Comment:
Maybe var suffixes can be in constant variable. I see these maybe times.

Reply:
I put it in constants file